src/containers/orgs/OrgsReducer.js

```diff
@@ -155,6 +168,31 @@ export default function orgsReducer(state :Map = INITIAL_STATE, action :Object)
       return state;
     }
 
+    case addConnection.case(action.type): {
+      const seqAction :SequenceAction = action;
+      return addConnection.reducer(state, action, {
+        REQUEST: () => state
+          .setIn([ADD_CONNECTION, 'requestState'], RequestStates.PENDING)
+          .setIn([ADD_CONNECTION, seqAction.id], seqAction),
+        SUCCESS: () => {
+          const storedSeqAction :SequenceAction = state.getIn([ADD_CONNECTION, seqAction.id]);
+          if (storedSeqAction) {
+            const { connection, organizationId } = storedSeqAction.value;
+            return state
+              .setIn([ADD_CONNECTION, 'requestState'], RequestStates.SUCCESS)
+              .updateIn(
+                ['orgs', organizationId, 'connections'],
```

Comment:
Should `requestState`, `orgs`, and `connections` be consts?

Reply:
I know you all do something different in other apps but I don't know how I feel about it. 

I agree. I don't like having these magic strings all over the place. However, I don't feel like this is any better:

```
import { REDUX_CONSTANTS } from '../../core/redux';
const { ORGS, CONNECTIONS } = REDUX_CONSTANTS;
...
const mapStateToProps = (state :Map) => ({
  requestStates: {
    [UPDATE_ROLE_GRANT]: state.getIn([ORGS, UPDATE_ROLE_GRANT, REQUEST_STATE]),
  }
});
```

Reply:
Yeah, I think the the benefit is that if you need to change that string for whatever reason, you can do it in one place. You could argue that that the const name could just as easily be misspelled as the string, but the at least linter would yell at you for that. You do you though! Just a suggestion.

Reply:
Solid points. I'll start playing with this approach going forward to see how it feels.